addons/mail/demo/discuss/public_channel_demo.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <record model="mail.guest" id="mail.public_guest">
```

Comment:
Better have more meaningful names in the xml ids, also they should have demo in them I believe. `guest_alex_demo` maybe?
I'm not sure if we want the model name at the start or at the end in this case.